cloud/gcp/templates/gke_autopilot/setup/main.tf

```diff
@@ -0,0 +1,83 @@
+# google_client_config and kubernetes provider must be explicitly specified like the following.
```

Comment:
**nitpick:** Is there a kubernetes provider in this file?

Reply:
Removed